src/MsSqlAcceptanceTests/MsSqlAcceptanceTests.csproj

```diff
@@ -7,7 +7,6 @@
     <AssemblyOriginatorKeyFile>$(SolutionDir)Test.snk</AssemblyOriginatorKeyFile>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.0.0" />
```

Comment:
This isn't redundant. As soon as you need to test on .NET Core, you'll need this, and you'll also need NUnit3TestAdapter.